api/dashboard_integration_test.go

```diff
@@ -2734,6 +2734,10 @@ func (s *PortalLinkIntegrationTestSuite) Test_GetPortalLinks_ValidPortalLinks()
 	// Act
 	s.Router.ServeHTTP(w, req)
 
+	if w.Code != http.StatusOK {
+		return
+	}
```

Comment:
I don't understand why this was added. It bypasses the following assertion which ensures the response is OK.